### PR TITLE
Update Docs: Configuration “configuration-options”

### PR DESCRIPTION
**Pull request by Zearin**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by Zearin** [comment]
(Wow, it really created a PR automatically based on my edits!  That is very cool.)

Just learning about this project, and thought I’d try and contribute a few small edits to the docs.

According to the docs style guide, documentation shouldn’t include latin abbreviations (like “e.g.” or “i.e.”), with the sole exception of “etc.”.  

Most of this PR is correcting this discrepancy, with a few other small copy-edits as well.

File: website/content/docs/configuration-options.md [hunk 1]
```diff
@@ -265,14 +266,14 @@ If you have an explicit frontmatter format declared, this option allows you to s
 
 ### `slug`
 
-For folder collections where users can create new items, the `slug` option specifies a template for generating new filenames based on a file's creation date and `title` field. (This means that all collections with `create: true` must have a `title` field (a different field can be used via [`identifier_field`](#identifier_field)).
```
Comment (martinjagodic) [inline website/content/docs/configuration-options.md:268]: Please keep the link

**Review by martinjagodic — CHANGES_REQUESTED** [review]

**Comment by privatemaker** [comment]
@Zearin I can also take this one too, if you'd like!!

**Comment by martinjagodic** [comment]
@Zearin @privatemaker I created the new repo at https://github.com/decaporg/decap-website, so this PR would have to be created again on that repo. Sorry for the mess, I kinda missed it.

**Pull request closed without merging**